### PR TITLE
[stats] stats own module/look cleaner

### PR DESCRIPTION
**Pull request by FragsAreUs**

### CODE REVIEW

**Comment by FragsAreUs** [comment]
lol but I dont have the unlock all in patches anymore it did not upload that change reee

File: src/client/module/stats.cpp [hunk 1]
```diff
@@ -29,29 +20,29 @@ class patches final : public module
...
-
-			});
+		});
 
 	}
-};
\ No newline at end of file
+};
+
+REGISTER_MODULE(stats);
```
Comment (momo5502) [inline src/client/module/stats.cpp:48]: Newline at the end would be nice

File: src/client/module/stats.cpp [hunk 2]
```diff
@@ -29,29 +20,29 @@ class patches final : public module
 	void patch_mp() const
 	{
 
-		// add Unlock all Command - only does level for squad member 1 if people want level 60 on all squad members let me know and I can just add the rest of the squad members
+		// add Unlock all Command
```
Comment (momo5502) [inline src/client/module/stats.cpp:23]: https://pbs.twimg.com/media/Eey7iwYX0AAwO_E.jpg

File: src/client/module/stats.cpp [hunk 3]
```diff
@@ -29,29 +20,29 @@ class patches final : public module
 	void patch_mp() const
 	{
```
Comment (momo5502) [inline src/client/module/stats.cpp:22]: No need for an empty line here

**Review by momo5502 — CHANGES_REQUESTED** [review]
Only small cosmetical stuff, otherwise it's fine

**Comment by FragsAreUs** [comment]
going to have to remake real quick as it keep saying its conflicting with patches when it should not be

**Comment by FragsAreUs** [comment]
in my branch i dont have the conflicting patches must of been something I messed up

**Pull request closed without merging**